fmatch/matcher.py

```diff
@@ -19,20 +19,24 @@
 
 class Matcher:
     """Matcher"""
-
+    # pylint: disable=too-many-instance-attributes
```

Comment:
@vishnuchalla Since you were the last to add codes to be skipped at [pylint.yml](https://github.com/cloud-bulldozer/py-commons/blob/main/.github/workflows/pylint.yml#L23) , How do you feel about adding this hint versus ignoring the code in pylint?